lib/opentoken-lib.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+
+/**
+ * @typedef opentokenLib
+ * @function createSignedTokenRequests
+ * @function createTokenRequest
+ */
+
+
+/**
+ * @typedef signedTokenRequests
+ * @property {SignedTokenRequest} private
+ * @property {SignedTokenRequest} public
+ */
+
+
+/**
+ * Creates an instance of opentokenLib.
+ *
+ * @param {string} host
+ * @param {string} logLevel
+ * @return {opentokenLib}
+ */
+module.exports = (host, logLevel) => {
+    var container, SignedTokenRequest, TokenRequest;
+
+    container = require("./container")(host, logLevel);
+    SignedTokenRequest = container.resolve("SignedTokenRequest");
+    TokenRequest = container.resolve("TokenRequest");
```

Comment:
Instead of this:

    container = require("...")(...);
    X = container.resolve("x");
    function a() { ... }
    return { a }

How about this?

    container = require("...")(...);
    return container.call((X) => {
        function a() { ... }
        return { a }
    })

Would it simplify the number of calls to `container.resolve()` by using `container.call()` instead?